spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are described using <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are case-insensitive and so are compared using ASCII case-insensitive comparisons.
+          Built-in time zones may be <dfn variants="named time zone">named time zones</dfn>, represented by the [[Identifier]] property of the records returned by AvailableNamedTimeZoneIdentifiers.
```

Comment:
I find the "built-in time zone" notion confusing. Are the built-in time zones exactly those that are usable within an implementation? Or are there non-built-in time zones that are usable?

This sentence seems overlapping with the "available named time zone identifier" notion below. Perhaps you can just use the word "available" instead of "built-in"?

Reply:
> Perhaps you can just use the word "available" instead of "built-in"?

Good idea, I'll make that change.

"Available" means the time zones that the implementation recognizes and can use for operations like GetNamedTimeZoneEpochNanoseconds and GetNamedTimeZoneOffsetNanoseconds. The term "built-in" was originally adopted from Temporal where there are custom time zone objects too. 

Given that Temporal isn't merged yet, I'll update this PR to use "available" everywhere. If changes are needed to support Temporal, we can figure that out when Temporal is ready for Stage 4.